qemu/tests/balloon_stress.py

```diff
@@ -25,35 +25,35 @@ def run(test, params, env):
     :param env: Dictionary with test environment.
     """
 
-    error.context("Boot guest with balloon device", logging.info)
+    error_context.context("Boot guest with balloon device", logging.info)
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
 
-    default_memory = int(params.get("default_memory", params['mem']))
     timeout = float(params.get("login_timeout", 360))
     session = vm.wait_for_login(timeout=timeout)
-    # for media player configuration
-    if params.get("pre_cmd"):
-        session.cmd(params.get("pre_cmd"))
 
     driver_name = params["driver_name"]
     utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
     balloon_test = BallooningTestWin(test, params, env)
 
-    error.context("Play video in guest", logging.info)
-    play_video_cmd = params["play_video_cmd"]
-    session.sendline(play_video_cmd)
-    # need to wait for wmplayer loading remote video
-    time.sleep(float(params.get("loading_timeout", 60)))
+    video_play = utils_misc.InterruptedThread(
+        utils_test.run_virt_sub_test, (test, params, env),
+        {"sub_type": params.get("sub_test")})
+    video_play.start()
+    # Wait for media player to load video
+    time.sleep(60)
```

Comment:
need around 30s in my host to install player and wake up video, I think 60s is ok as some machine maybe slow